mesos-rxjava-example/mesos-rxjava-example-framework/src/main/java/org/apache/mesos/rx/java/example/framework/sleepy/Sleepy.java

```diff
@@ -203,7 +219,7 @@ private static TaskInfo sleepTask(@NotNull final AgentID agentId, @NotNull final
                     .setEnvironment(Environment.newBuilder()
                         .addVariables(
                             Environment.Variable.newBuilder()
-                                .setName("SLEEP_SECONDS").setValue("15")
+                                .setName("SLEEP_SECONDS").setValue(sleepSeconds)
```

Comment:
The code is the exact same shape, but it's dealing with different versions of mesos protos :(

We might be able to come up with something to try and keep it in sync easier in the future.